frontend/src/features/settings/components/dashboard/ManageSettingsDashboard.tsx

```diff
@@ -25,15 +25,15 @@ export const ManageSettingsDashboard = React.memo(() => {
     companyId as number,
   );
 
-  const isStaffActingAsCompany = Boolean(idirUserDetails?.userAuthGroup);
+  const isStaffActingAsCompany = Boolean(idirUserDetails?.userRole);
   const isFinanceUser =
-    idirUserDetails?.userAuthGroup === IDIR_USER_AUTH_GROUP.FINANCE;
+    idirUserDetails?.userRole === IDIR_USER_AUTH_GROUP.FINANCE;
```

Comment:
Consider renaming the enum "IDIR_USER_AUTH_GROUP" to "IDIR_USER_ROLES", as mentioned earlier